projects/go-lib/src/lib/components/go-table/go-table.component.html

```diff
@@ -123,18 +110,43 @@ <h3 class="go-heading-3 go-heading--no-margin go-table-header__title" *ngIf="tab
               </ng-template>
             </td>
           </tr>
+
+          <!-- Child Rows Template -->
+          <ng-container *ngIf="childRows && item.showDetails">
+            <tr *ngFor="let childItem of item[childRowsKey]">
+              <td class="go-table__body-column">
+                <span
+                  class="go-table__child-row-indicator"
+                  [ngStyle]="{ 'border-left-color': brandColor }">
+                </span>
+              </td>
+              <td
+                *ngIf="localTableConfig.selectable"
+                class="go-table__body-column"
+                width="60">
+              </td>
+              <td
+                *ngFor="let col of childRowColumns"
+                class="go-table__body-column"
+                [ngStyle]="{ 'vertical-align': col.alignment }">
+                <ng-container *ngIf="!col.goTableCell; else childColumnCellOutlet">
+                  {{ col.getFieldData(childItem) }}
+                </ng-container>
```

Comment:
If the specified column does not have a custom template declared, then we output the value for the column.